pkg/crc/cluster/cluster.go

```diff
@@ -214,8 +215,9 @@ func addProxyCACertToCluster(sshRunner *ssh.Runner, ocConfig oc.Config, proxy *n
   }
 }
 `
-	// Replace the carriage return with `\n` char
-	p := fmt.Sprintf(proxyCABundleTemplate, strings.ReplaceAll(proxy.ProxyCACert, "\n", `\n`), trustedCAName)
+	// Replace the carriage return ("\n" or "\r\n") with `\n` char
```

Comment:
I realise now the comment is confusing? This replaces the carriage return with the litteral `\n` string (aka "\\n"),  not with a single 'char'. Could be a good opportunity to fix the comment.